docs/tutorial/Pre-requisites of running a Waku node.md

```diff
@@ -0,0 +1,116 @@
+---
+title: 'Pre-requisites of running a Waku node'
+---
+
+Pre-requisites of running a Waku node
+===
+
+## Table of Contents
+
+[TOC]
```

Comment:
Same for this part, you can safely remove them. I'll add an explanation when revising this doc with how to run rln-relay chat2. 
```suggestion

```